src/Machete/Values/EntityValueList.cs

```diff
@@ -51,6 +51,9 @@ public bool TryGetValue(int index, out Value<TValue> value)
                 index >= _values.Count && _slice.TryGetSlice(i, out textSlice);
                 i++)
             {
+                if (textSlice.SourceSpan.IsEmpty)
```

Comment:
K, however, the problem is that it is saying things have an addition slice that is simply not there right now. Perhaps the fix is in the TrySlice method itself?